content/fr/about/contact.md

```diff
@@ -28,9 +28,9 @@ type: wide
         <a class="twitter-timeline" data-height="500" data-dnt="true" href="https://twitter.com/InnerSourceOrg?ref_src=twsrc%5Etfw"></a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
       </div>
       <div class="col-md-5 offset-md-1">
-        <p class="h2 section-title">Stay in Touch</p>
-        <p class="mb-4">Most of the action in the community happens on our <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">Slack Channel</a>. You can also keep informed about the InnerSource Commons news, events and activities by signing up to our mailing list below, follow us on <a href="https://twitter.com/InnerSourceOrg">Twitter</a> and <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, and subscribing to our <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube Channel</a>.</p>
-        <p class="h3 section-title">Sign up to our mailing list</p>
+        <p class="h2 section-title">Rester en contact</p>
+        <p class="mb-4">La plupart des actions de la communauté se déroulent sur notre <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">Canal Slack</a>. Vous pouvez également vous tenir au courant des nouvelles, des événements et des activités d'InnerSource Commons en vous inscrivant à notre liste de diffusion ci-dessous, et en nous suivant sur <a href="https://twitter.com/InnerSourceOrg">Twitter</a> et <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, et souscrire à notre <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Chaine YouTube</a>.</p>
```

Comment:
Chaîne

Reply:
Same answer as previously, will keep the "chaine" version

Reply:
I think this is ok to keep uppercase for Chaine and Canal because this is a basically what has been done for EN version, and look & feel is better since it does correspond to a Web link